api/src/main/proto/spine/protodata/insertion.proto

```diff
@@ -61,7 +61,7 @@ message InsertedPoints {
     option (entity).kind = VIEW;
 
     // The relative path to the code file.
```

Comment:
I would append the notion of possible "relativeness" to the definition of `File`.